src/components/Chat/InputMessageV2.tsx

```diff
@@ -158,7 +345,7 @@ const InputMessageV2 = ({
 
     Alert.alert(
       '',
-      type === 'ANONYMOUS'
+      type === ANONYMOUS
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `type` comparison in the alert message has been updated to use a constant instead of a string literal. This is a good practice as it reduces the chance of typos causing bugs.



<!-- This is an auto-generated comment by OSS CodeRabbit -->